pkg/apis/push/v1alpha1/pushapplication_types.go

```diff
@@ -0,0 +1,63 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// PushApplicationSpec defines the desired state of PushApplication
+// +k8s:openapi-gen=true
+type PushApplicationSpec struct {
+
+	// Description is a description of the app to be displayed in
+	// the UnifiedPush Server admin UI
+	Description string `json:description`
+
+	// Developer is a string containing the name of the developer
+	// that created the app. It is displayed in the UnifiedPush
+	// Server admin UI
+	Developer string `json:developer`
```

Comment:
@secondsun is this field optional --> https://github.com/aerogear/aerogear-unifiedpush-server/blob/ff5ba2cb8446d929e5fd426bcb645299b4559914/model/api/src/main/java/org/jboss/aerogear/unifiedpush/api/PushApplication.java#L48 ?

Reply:
It looks like when creating an application from the UI currently, that it's not exposed and a default of `developer` is set:

![image](https://user-images.githubusercontent.com/442386/58549375-4a070e80-8203-11e9-9b9e-a5365b45df5f.png)

I haven't looked into whether or not that's the UI setting that or if it's a default in the API. I guess the UI that we present in OpenShift might just set a default since it might be difficult to get the openshift user name?

I'm happy to remove it from here if you think that's best, or make it optional here?

Reply:
Let's remove it! It would mean less confusion if we don't have these kind of rarely used stuff.

Reply:
Ok cool, I'll remove it now so. We can add it as optional in future if we want to :+1:

How do the other fields look to you?

Reply:
Didn't see anything strange :)